src/shared/components/FileStep/FileStep.style.ts

```diff
@@ -0,0 +1,114 @@
+import { colors, sizes, transitions, typography } from '@/shared/theme'
+import styled from '@emotion/styled'
+import CircularProgressbar from '../CircularProgressbar'
+import Text from '../Text'
+
+type StepProps = {
+  active?: boolean
+  disabled?: boolean
+}
+
+export const StepWrapper = styled.div<StepProps>`
```

Comment:
It seems there are some issues with sizing here:

1) The height of the step changes once I pick the file. It should be fixed
<img width="909" alt="Screenshot 2021-03-09 at 13 13 17" src="https://user-images.githubusercontent.com/12646744/110468949-493e2480-80d9-11eb-9d6b-f25959dd8b50.png">
<img width="908" alt="Screenshot 2021-03-09 at 13 13 04" src="https://user-images.githubusercontent.com/12646744/110468957-4b07e800-80d9-11eb-93e7-db4617d6f829.png">

2) Seems we're not matching the height from the design, let's try to do that